packages/mobile/src/identity/verification.ts

```diff
@@ -346,6 +343,7 @@ export function* doVerificationFlow(withoutRevealing: boolean = false) {
       }
     }
 
+    ValoraAnalytics.track(VerificationEvents.verification_reveal_all_attestations_complete)
```

Comment:
Moving it all the way down here is a bit misleading because IIUC, the complete attestations request above is another place where transactions need to be submitted. Moving this line 327 might be better. I might also support the idea of creating new events, even though we won't be able to correlate them with the one being sent today (changing the meaning of the events means we should compare `verification_reveal_all_attestations_complete` events before and after this change anyway.)

Reply:
Oh, also I think where this is here, it would give the same information as the `verification_complete` event. I'd have to verify that though.

Reply:
that's a good point -- i was trying to move it to where it was before, but we should probably just add `verification_complete` to the funnel as the true terminating state